src/building/how-to-build-and-run.md

```diff
@@ -6,15 +6,23 @@ be hacking on `rustc`, you'll want to tweak the configuration of the compiler.
 The default configuration is oriented towards running the compiler as a user,
 not a developer.
 
+## Get the source code
+
+The very first step to work on `rustc` is to clone the repository:
+
+```text
```

Comment:
It'd be nice that we had the same annotation as well as the below.

```suggestion
```bash
```

Reply:
Ah right, wanted to comment on that.
I did that first but it gave some weird looking coloring on `clone` (but the rest stayed uncolored, so I opted for `text` instead.

Obviously semantically that's not the right thing, but UI wise it felt better.
But happy to change that back to `bash`

Reply:
It's the syntax highlighting for bash, like: https://rustc-dev-guide.rust-lang.org/profiling/with_perf#gathering-a-perf-profile-from-a-perfrust-langorg-test
I think it's fine here since we use it in other places.